src/main/java/lotto/Application.java

```diff
@@ -1,7 +1,17 @@
 package lotto;
 
+import lotto.controller.LottoController;
+import lotto.service.LottoNumberGenerator;
+import lotto.service.LottoService;
+import lotto.service.StatisticsService;
+
 public class Application {
     public static void main(String[] args) {
-        // TODO: 프로그램 구현
+        LottoNumberGenerator generator = new LottoNumberGenerator();
+        LottoService lottoService = new LottoService(generator);
+        StatisticsService statisticsService = new StatisticsService();
+        LottoController controller = new LottoController(lottoService, statisticsService);
```

Comment:
```suggestion
        LottoController controller = new LottoController(new LottoService(), new StatisticsService());
```

저라면 로또서비스에 generator를 포함할 것 같은데, 구분하신 이유가 궁금해요!
서비스라면 로또 도메인을 관리할텐데, 로또는 랜덤과 가까워서 함께 있어도 된다고 생각헀어요

Reply:
저는 확장성을 고려하였을 때, generator의 방식이 바뀔 수 있기 때문에 이런 방식으로 외부에서 주입해주는 것도 좋다고 생각합니다! 만약 더 생각해본다면, 인터페이스를 활용하는 방법이 있을 수 있겠네요!